File: Python/isSubsequence.py

s = "abc" 
t = "ahbgdc"
def isSubsequence (s,t):
    x = 0
    for str in t:
        if x < len(s) and str == s[x]:
            x += 1
    return x == len(s)      
        
print(isSubsequence(s,t))    

File: JavaScript/isSubsequence.js

let s = "abc" 
let t = "ahbgdc"


var isSubsequence = function(s, t) {
    
    let x = 0
    
    for (let str of t) {

        if (x < s.length && str == s[x]) x++

    }
return x == s.length
};


console.log(isSubsequence(s,t))